### PR TITLE
[lldb] Fixed the TestNetBSDCore test

### PR DESCRIPTION
**Pull request by slydiman**
TestNetBSDCore.py contains 3 classes with the same test names test_aarch64 and test_amd64. It causes conflicts because the same build dir. Add suffixes to avoid conflicts.

The error message on the Windows host running with `-j 2` is the following:
```
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process:
'E:\\projects\\lldb\\build-lldb\\lldb-test-build.noindex\\functionalities\\postmortem\\netbsd-core\\TestNetBSDCore.test_aarch64\\Incomplete.log'
```

### CODE REVIEW

File: lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py [hunk 1]
```diff
@@ -147,12 +147,12 @@ def check_stack(self, process, pid, filename):
         self.check_backtrace(thread, filename, backtrace)
 
     @skipIfLLVMTargetMissing("AArch64")
-    def test_aarch64(self):
+    def test_aarch64_A(self):
```
Comment (JDevlieghere) [inline lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py:150]: Can we give this a more meaningful name, like `test_aarch64_single` or `test_aarch64_single_threaded`? Or maybe keep these and only rename the ones below.

File: lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py [hunk 2]
```diff
@@ -177,12 +177,12 @@ def check_stack(self, process, pid, filename):
         self.assertEqual(thread.GetStopReasonDataAtIndex(0), 0)
 
     @skipIfLLVMTargetMissing("AArch64")
-    def test_aarch64(self):
+    def test_aarch64_B(self):
```
Comment (JDevlieghere) [inline lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py:180]: `test_aarch64_thread_signaled`?

File: lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py [hunk 3]
```diff
@@ -207,11 +207,11 @@ def check_stack(self, process, pid, filename):
         self.assertEqual(thread.GetStopReasonDataAtIndex(0), signal.SIGSEGV)
 
     @skipIfLLVMTargetMissing("AArch64")
-    def test_aarch64(self):
+    def test_aarch64_C(self):
```
Comment (JDevlieghere) [inline lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py:210]: `test_aarch64_process_signaled`?

File: lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py (hunk 1 above)
Comment (slydiman) [reply lldb/test/API/functionalities/postmortem/netbsd-core/TestNetBSDCore.py]: Sure. I have renamed them to suggested names. Thanks.

**Review by JDevlieghere — APPROVED** [review]
LGTM

**Pull request merged**